changelog/generated/v1.5.2.md

```diff
@@ -0,0 +1,3 @@
+## v1.5.2
+
+No changes for this release!
```

Comment:
Can you add a note about this being a security release and to bump images from previous v1.5 to v1.5.2